src/data-tutorials/users.tsx

```diff
@@ -866,6 +866,20 @@ const Users: User[] = [
     website: 'blog/tutorials/how-to-quickly-build-an-nft-collection-on-celo',
     tags: ['beginner','foundation', 'nft', 'foundation'],
   },
+  {
+    title: 'How should you use Celo Blockchain',
```

Comment:
Can I know why it is changing two articles

Reply:
I initially had a git conflict there. And I notice it was pointing at those so I remove one that was duplicated